manager/config/network-policies/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+resources:
+- authorization.yaml
```

Comment:
I suggest renaming `authorization.yaml` to `connectors-ingress.yaml` to give it the right context (the fact that it's in `network-policies` folder fills the rest of the context)